pages/games/[chainLabel]/[gameId].tsx

```diff
@@ -540,45 +547,44 @@ function GamePage({
   );
 }
 
-const GameActions: React.FC<StackProps> = ({ ...props }) => {
-  const { isMaster } = useGame();
+type QueryParams = { gameId: string; chainLabel: string };
 
-  const { openActionModal } = useGameActions();
-  const itemsCatalogModal = useDisclosure();
+export const getStaticPaths: GetStaticPaths = async () => {
+  const paths: { params: QueryParams }[] = [];
 
-  return (
-    <VStack
-      h="100%"
-      bg="cardBG"
-      px={{ base: 4, sm: 8 }}
-      py={8}
-      align="stretch"
-      spacing={4}
-      {...props}
-    >
-      <Button onClick={itemsCatalogModal.onOpen} size="sm">
-        show items catalog
-      </Button>
-      {isMaster && (
-        <>
-          <Button
-            onClick={() => openActionModal(GameMasterActions.CREATE_ITEM)}
-            size="sm"
-          >
-            create Item
-          </Button>
-          <Button
-            onClick={() => openActionModal(GameMasterActions.CREATE_CLASS)}
-            size="sm"
-          >
-            create Class
-          </Button>
-        </>
-      )}
-      <ItemsCatalogModal
-        isOpen={itemsCatalogModal.isOpen}
-        onClose={itemsCatalogModal.onClose}
-      />
-    </VStack>
+  await Promise.all(
+    SUPPORTED_CHAINS.map(async chain => {
+      const chainLabel = getChainLabelFromId(chain.id);
+      if (!chainLabel) {
+        return;
+      }
+      const games = await getGamesForChainId(chain.id);
+
+      paths.push(
+        ...games.map(game => ({
+          params: {
+            chainLabel,
+            gameId: game.id,
+          },
+        })),
+      );
+    }),
   );
+
+  return { paths, fallback: true };
+};
+
+export const getStaticProps: GetStaticProps = async context => {
+  const chainLabel = context.params?.chainLabel as string;
+  const gameId = context.params?.gameId as string;
+  const chainId = getChainIdFromLabel(chainLabel);
+  const game =
+    !!chainId && !!gameId ? await getGameForChainId(chainId, gameId) : null;
+
+  return {
+    props: {
+      game,
+    },
+    revalidate: 60,
```

Comment:
Should we revalidate add the end of each transaction flow? Getting some character image / equipped item mismatches when I equip and unequip an item

Reply:
Actually, after testing more, I can't get this issue to appear again... maybe it was a temporary issue with my comp. Will go and merge!